src/components/Link/index.tsx

```diff
@@ -125,6 +126,7 @@ const Link = forwardRef(
       rel,
       className,
       onClick,
+      'aria-label': ariaLabel,
```

Comment:
> for links, buttons and clickable elements, the text content of the element typically provides the label

as children is mandatory that should be enough no ?

Reply:
@chambo-e Here is a good example why it is not enough sometimes:

![Screenshot 2022-11-30 at 11 19 13](https://user-images.githubusercontent.com/15812968/204770449-858a2eff-7357-4b3b-a6c3-0fd08790eafc.png)

https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA8

Reply:
Yeah but we don't do that right ? 🤔

Reply:
@chambo-e we actually do with learn more links for example